packages/core/src/lib/toast/__tests__/toast-banner.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { describe, it, expect, vi } from 'vitest';
+import { fireEvent, render, screen } from '@testing-library/svelte';
+
+import Toast from './toast-banner.spec.svelte';
+import { cxTestArguments, cxTestResults } from '$lib/__tests__/cx-test';
+import { ToastVariant } from '../variants';
+
+describe('Toast', () => {
+  it('Renders toast element with appropriate message, and action text', () => {
+    const { container } = render(Toast, {
+      variant: ToastVariant.Success,
+    });
+
+    expect(screen.getByText('This is the message.')).toBeVisible();
+    expect(screen.getByText('This is the action button.')).toBeVisible();
+    expect(container.querySelector('.text-success-dark')).toBeVisible();
+    expect(screen.getByRole('button')).toBeVisible();
+  });
+
+  it('Calls the dismiss function when the close button is clicked', async () => {
+    const dismissMock = vi.fn();
```

Comment:
[Nit] The "mock" in the name here is a bit redundant; it's defined with `vi.fn()`, we know what it is. `dismiss` is a little easier to read/use